components/Ref/Ref.tsx

```diff
@@ -65,7 +65,7 @@ function Ref(props: Props) {
                                 <hr />
                             </div>
                         ) : null}
-                        <p>Share your Ref-Link with people which might find Cofnet useful.</p>
+                        <p>Share your referral Link with people which might find CoflNet useful.</p>
```

Comment:
Dosnt "Referral-Link" or "Referral Link" look better?

Reply:
Referral-Link is grammatically wrong, but in my opinion Referral Link looks better